nginx/timings.py

```diff
@@ -68,7 +69,9 @@ def to_tags(self, tag_whitelist, **kwargs):
 APDEX_THRESHOLDS = (3, 12)
 
 
-def parse_nginx_apdex(logger, line):
+def parse_nginx_apdex(*args):
```

Comment:
why not just add the newer args? I don't see the need to maintain backward compatibility